package.json

```diff
@@ -99,8 +99,8 @@
     "po-loader": "^0.3.1",
     "po2json": "^0.4.2",
     "prettier": "^1.15.3",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
[CVE-2018-6341](https://nvd.nist.gov/vuln/detail/CVE-2018-6341) says that this vulnerability was fixed in `16.2.1` and `16.3.3`?